.github/workflows/storybook.yml

```
@@ -0,0 +1,22 @@
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                node-version: lts/*
+            - run: npm ci
+            - run: npm run storybook:build
+            - uses: JamesIves/github-pages-deploy-action@v4
+              with:
+                folder: 'storybook-static'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)